test/scales/modifiedLogScaleTests.ts

```diff
@@ -2,187 +2,241 @@
-        assert.closeTo(scale.scale(x), Math.log(x) / Math.log(10), 0.1);
+      beforeEach(() => {
+        scale = new Plottable.Scales.ModifiedLog(base);
       });
-    });
 
-    it("x = invert(scale(x))", () => {
-      [0, 1, base, 100, 0.001, -1, -0.3, -base, base - 0.001].forEach((x) => {
-        assert.closeTo(x, scale.invert(scale.scale(x)), epsilon);
-        assert.closeTo(x, scale.scale(scale.invert(x)), epsilon);
+      it("is an increasing, continuous function that can go negative", () => {
+        d3.range(-base * 2, base * 2, base / 20).forEach((x: number) => {
+          // increasing
+          assert.operator(scale.scale(x - epsilon), "<", scale.scale(x));
+          assert.operator(scale.scale(x), "<", scale.scale(x + epsilon));
+          // continuous
+          assert.closeTo(scale.scale(x - epsilon), scale.scale(x), epsilon);
+          assert.closeTo(scale.scale(x), scale.scale(x + epsilon), epsilon);
+        });
+        assert.closeTo(scale.scale(0), 0, epsilon);
       });
-    });
 
-    it("domain defaults to [0, base]", () => {
-      scale = new Plottable.Scales.ModifiedLog(base);
-      assert.deepEqual(scale.domain(), [0, base], "default domain is [0, base]");
-    });
-
-    it("can be padded", () => {
-      scale.addIncludedValuesProvider((scale: Plottable.Scales.ModifiedLog) => [0, base]);
-      scale.padProportion(0);
-      let unpaddedDomain = scale.domain();
-      scale.padProportion(0.1);
-      assert.operator(scale.domain()[0], "<", unpaddedDomain[0], "left side of domain has been padded");
-      assert.operator(unpaddedDomain[1], "<", scale.domain()[1], "right side of domain has been padded");
-    });
+      it("has log() behavior at values > base", () => {
+        [10, 100, 23103.4, 5].forEach((x) => {
+          assert.closeTo(scale.scale(x), Math.log(x) / Math.log(10), 0.1);
+        });
+      });
 
-    it("autoDomain() expands single value correctly", () => {
-      scale.padProportion(0);
-      let singleValue = 15;
-      scale.addIncludedValuesProvider((scale: Plottable.Scales.ModifiedLog) => [singleValue, singleValue]);
-      assert.deepEqual(scale.domain(), [singleValue / base, singleValue * base],
-        "positive single-value extent was expanded to [value / base, value * base]");
-      singleValue = -15;
-      scale.autoDomain();
-      assert.deepEqual(scale.domain(), [singleValue * base, singleValue / base],
-        "negative single-value extent was expanded to [value * base, value / base]");
-      singleValue = 0;
-      scale.autoDomain();
-      assert.deepEqual(scale.domain(), [-base, base],
-        "zero single-value extent was expanded to [base, -base]");
-    });
+      it("ensures x = invert(scale(x))", () => {
+        [0, 1, base, 100, 0.001, -1, -0.3, -base, base - 0.001].forEach((x) => {
+          assert.closeTo(x, scale.invert(scale.scale(x)), epsilon);
+          assert.closeTo(x, scale.scale(scale.invert(x)), epsilon);
+        });
+      });
```

Comment:
Why not? We need to test that the `modifedLogScale` satisfies the contract of a `quantitativeScale`. If we have a mistake in either our `scale()` or `invert()` method, then this test will catch that.